ansys/dpf/core/collection.py

```diff
@@ -59,9 +61,43 @@ def __init__(self, dpf_type, collection=None, server: server.DpfServer = None):
         elif hasattr(collection, "_message"):
             self._message = collection._message
             self._collection = collection  # keep the base collection used for copy
+
         else:
             self._message = collection
 
+        if self._type == None:
+            self._type = types(int(self._message.type) + 1)
+
+    @staticmethod
+    def integral_collection(inpt, server: server.DpfServer = None):
+        """Creates a collection of integral type with a list.
```

Comment:
Short summary and long description should have a single line between.
```suggestion
        """Creates a collection of integral type with a list.

```